packages/core/tsup.config.ts

```diff
@@ -25,27 +31,32 @@ export default defineConfig({
                         args.path,
                         "utf8",
                     );
+                    let finalContents: string = contents;
 
-                    const lodashImportRegex =
-                        /import\s+?(?:(?:(?:[\w*\s{},]*)\s+from\s+?)|)[\'\"](?:(?:lodash\/?.*?))[\'\"][\s]*?(?:;|$|)/g;
                     const extension = path.extname(args.path).replace(".", "");
-
                     const loader = JS_EXTENSIONS.has(extension)
                         ? "jsx"
                         : (extension as any);
 
+                    // find and replace lodash imports
+                    const lodashImportRegex =
+                        /import\s+?(?:(?:(?:[\w*\s{},]*)\s+from\s+?)|)[\'\"](?:(?:lodash\/?.*?))[\'\"][\s]*?(?:;|$|)/g;
+
                     const lodashImports = contents.match(lodashImportRegex);
-                    if (!lodashImports) {
-                        return {
-                            loader,
-                            contents,
-                        };
+                    if (lodashImports) {
+                        finalContents = contents.replace("lodash", "lodash-es");
                     }
 
-                    const finalContents = contents.replaceAll(
-                        "lodash",
-                        "lodash-es",
-                    );
+                    // find version and replace
+                    const versionRegex = /const REFINE_VERSION = "\d.\d.\d";/gm;
```

Comment:
```suggestion
                    const versionRegex = /const REFINE_VERSION = "\d.\d.\d";/gm;
```
Should we consider this part as a separate plugin?